WcaOnRails/app/models/team.rb

```diff
@@ -8,6 +8,7 @@ class Team < ApplicationRecord
   scope :with_hidden, -> { unscope(where: :hidden) }
 
   scope :official, -> { where(id: Team.all_official.map(&:id)) }
+  scope :officers, -> { where(id: Team.all_officers.map(&:id)) }
```

Comment:
Is this scope used anywhere?

Reply:
Oh, my intention was to use it in [line 38 of the user model](https://github.com/thewca/worldcubeassociation.org/pull/3808/files#diff-572f7c99c29e70c2f7cbb9c219270579R38). I will update it to use it now! However, if you think it's best to not have the scope at all, I can undo the change and remove this line.